### PR TITLE
UHF-10239: Adds image gallery paragraph

### PR DESCRIPTION
**Pull request by annadruid**
# [UHF-10239](https://helsinkisolutionoffice.atlassian.net/browse/UHF-10239)

## What was done

* Configurations for the new image gallery paragraph
* New image style configurations required by the image gallery paragraph
* Translations for the image gallery paragraph validation
* Added a missing translation for hero validation which I noticed while adding image gallery translations.

## How to install
* Make sure your instance is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh`
* Update the Helfi Platform config
    * `composer require drupal/helfi_platform_config:dev-UHF-10239`
* Run `make drush-updb drush-cr`

## How to test

* [ ] The functionality testing guidance is in HDBT pr
* [ ] Check that code follows our standards

## Continuous documentation

* [ ] This feature has been documented/the documentation has been updated
* [ ] This change doesn't require updates to the documentation

## Translations

* [x] Translations have been added to .po -files and included in this PR

## Other PRs

* https://github.com/City-of-Helsinki/drupal-hdbt/pull/1110
* https://github.com/City-of-Helsinki/drupal-hdbt-admin/pull/310

[UHF-10239]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-10239?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by tuutti — CHANGES_REQUESTED** [review]

File: modules/hdbt_admin_tools/src/Plugin/Validation/Constraint/ImageGalleryConstraint.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?php
+
```
[the remark this answers is not in the document]
Comment (annadruid) [reply modules/hdbt_admin_tools/src/Plugin/Validation/Constraint/ImageGalleryConstraint.php:2]: Fixed!

File: modules/hdbt_admin_tools/src/Plugin/Validation/Constraint/ImageGalleryConstraintValidator.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<?php
```
[the remark this answers is not in the document]
Comment (annadruid) [reply modules/hdbt_admin_tools/src/Plugin/Validation/Constraint/ImageGalleryConstraintValidator.php:1]: Fixed!

File: modules/hdbt_admin_tools/src/Plugin/Validation/Constraint/ImageGalleryConstraintValidator.php [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+/**
+ * Validates the Image Gallery constraint.
+ */
+class ImageGalleryConstraintValidator extends ConstraintValidator {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function validate(mixed $value, Constraint $constraint) {
```
[the remark this answers is not in the document]
Comment (annadruid) [reply modules/hdbt_admin_tools/src/Plugin/Validation/Constraint/ImageGalleryConstraintValidator.php]: Fixed!

**Review by tuutti — APPROVED** [review]

**Review by teroelonen — CHANGES_REQUESTED** [review]
Pending the image style changes that I wrote to you on chat. Otherwise looks very good to me!

**Comment by annadruid** [comment]
> Pending the image style changes that I wrote to you on chat. Otherwise looks very good to me!

The image styles have now been updated!

**Review by teroelonen — APPROVED** [review]
Really great work with this one! 🦖

**Pull request merged**